uint256.go

```diff
@@ -508,6 +508,10 @@ func udivrem(quot, u []uint64, d *Int) (rem Int) {
 		}
 	}
 
+	if uLen < dLen {
```

Comment:
from https://github.com/holiman/uint256/pull/111#issuecomment-1067939986 ,so it should be
```
        if uLen < dLen {
		copy(rem[:], u[:])
		return rem
	}
```

Reply:
Except we need to handle `u[:]` more carefully here because it may be bigger than `Int`. Is `rem Int` all zeros at function start?

Reply:
Yes, `rem` is initialized blank.